easybuild/easyblocks/h/hdf5.py

```diff
@@ -81,7 +91,14 @@ def configure_step(self):
 
         super(EB_HDF5, self).configure_step()
 
-    # default make and make install are ok
+    # default make and make install are ok but add a pkconfig file
+    def install_step(self):
+        """Custom install step for HDF5"""
+        super(EB_HDF5, self).install_step()
+        write_file(os.path.join(self.installdir, "lib", "pkgconfig", "hdf5.pc"),
+                   HDF5_PKG_CONFIG % (self.version, os.path.join(self.installdir, "include"),
+                                      os.path.join(self.installdir, "lib")),
+                   forced=True)
```

Comment:
@akesandgren This is a bit... messy. How about:

```python
inc_dir = os.path.join(self.installdir, 'include')
lib_dir = os.path.join(self.installdir, 'lib')
hdf5_pc_txt = HDF5_PKG_CONFIG % (self.version, inc_dir, lib_dir)
write_file(os.path.join(lib_dir, 'pkgconfig', 'hdf5.pc'), hdf5_pc_txt, forced=True)
```

Can you clarify the `forced=True`?

Reply:
Just picked forced=True because i felt like it. Want me to remove it?

Reply:
If there's no reason to use `forced=True`, then yes, remove it (but that's already done I see). :)